### DISCUSSION TITLE
queries of derieved tables

### DISCUSSION THREAD

**Discussion by seymaaltay** [MIT-LCP/mimic-code#1220] [MIMIC-General]
how can I find the queries of derieved tables which shared in BigQuery?

**Comment by alistairewj** [comment]
For MIMIC-IV, they are all here: https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iv/concepts
Similarly for MIMIC-III: https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iii/concepts

I added some info to the README since it looks like we never spelled that out.

**Reply by liedezhanghu** [reply]
Could you please tell me how to use the concepts.I have finished all the steps follow https://github.com/MIT-LCP/mimic-code/tree/main/mimic-iii/concepts,and generated 50 tables, but how to use it?

**Reply by alistairewj** [reply]
After that, you use them as you would SQL tables.

If you're not sure how to use SQL, you could load them into Python with a library such as pandas: https://pandas.pydata.org/docs/reference/api/pandas.read_sql_table.html

Or export the tables from the database into a CSV, at which point you can use any program. Something like:
`\copy mimic_derived.sepsis TO 'sepsis.csv' CSV HEADER;`
... in psql would work.

**Reply by liedezhanghu** [reply]
I also want to ask a question that I don't understand. For example, for a patient with subject ID '56', we query the DOB field of table PATIENTS, and the date of birth is '1804-01-02 00:00:00'. However, using the record with subject ID '56', we query the table ICUSTAYS and find that the field of  intime is '2104-01-02 02:02:39'. If I calculate its age with these two values, it is obviously unreasonable that its age is more than 300 years old. There are many similar situations. So we want to know whether the subject ID is indeed the unique identification of the patient, or whether our method of counting age is correct?

**Reply by alistairewj** [reply]
https://mimic.mit.edu/docs/iii/about/time/#dates-of-birth

**Reply by liedezhanghu** [reply]
Thanks！

**Reply by liedezhanghu** [reply]
I read your article about extracting echocardiography data from mimiciii written six years ago(https://github.com/MIT-LCP/echo-data). After reading the readme part in detail, I still have doubts about how to extract it. First of all, https://physionet.org/users/ this link can't be accessed, secondly,where can I get the"Echocardiography data" CSV file?Please forgive my grammar mistakes because English is not my mother language.

**Reply by alistairewj** [reply]
for new discussion items please create a new discussion topic!

**Reply by liedezhanghu** [reply]
Yes， sir， I will create a new discussion topic.I am eagerly  looking forward to your reply.
